stripe/res/layout/layout_paymentsheet_add_card_item.xml

```diff
@@ -9,11 +8,14 @@
     <com.google.android.material.card.MaterialCardView
         android:id="@+id/card"
         style="@style/StripePaymentOptionItemCard"
-        app:layout_constraintTop_toTopOf="parent"
+        android:duplicateParentState="true"
```

Comment:
Is this to avoid having to manually propagate the enabled state?

Reply:
Yes, this actually only works for `selected`. Still need to manually set `enabled`.